### PR TITLE
[Validator] Add missing DE validator translations

### PR DESCRIPTION
**Pull request by janvt**
| Q             | A
| ------------- | ---
| Branch       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #30156 
| License       | MIT

Updated the validator translation file with the missing German (DE) translations.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf [hunk 1]
```diff
@@ -322,6 +322,14 @@
                 <source>This is not a valid UUID.</source>
                 <target>Dies ist keine gültige UUID.</target>
             </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Dieser Wert sollte ein Multiplikator von {{ compared_value }} sein.</target>
```
Comment (derrabus) [inline src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]: 🇺🇸 a multiple of
🇩🇪 ein Vielfaches von

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf [hunk 2]
```diff
@@ -322,6 +322,14 @@
...
+            </trans-unit>
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Dieser BIC (internationaler Bankleitzahl) ist nicht mit der IBAN {{ iban }} assoziiert.</target>
```
Comment (derrabus) [inline src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]: Grammar: "internationale" (without "r" at the end).

Also, I would rather move "BIC" into the brackets: "Diese internationale Bankleitzahl (BIC)…"

**Review by derrabus — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf (hunk 1 above)
Comment (janvt) [reply src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]: Yeah, ok, that sounds better.

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf (hunk 2 above)
Comment (janvt) [reply src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]: Articles & conjugation. Still my weak spot after 20 years of speaking German :D

**Review by derrabus — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Review by dmaicher — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @janvt.

**Pull request merged**